src/Microsoft.CodeQuality.Analyzers/Core/QualityGuidelines/MarkMembersAsStatic.cs

```diff
@@ -161,7 +170,8 @@ private static bool ShouldAnalyze(IMethodSymbol methodSymbol, Compilation compil
             // If this looks like an event handler don't flag such cases.
             if (methodSymbol.Parameters.Length == 2 &&
                 methodSymbol.Parameters[0].Type.SpecialType == SpecialType.System_Object &&
-                IsEventArgs(methodSymbol.Parameters[1].Type, compilation))
+                IsEventArgs(methodSymbol.Parameters[1].Type, compilation) &&
+                !methodSymbol.IsAccessorMethod())
```

Comment:
❓ Which test covers this? It wasn't clear to me from reading why we'd need this.

Reply:
Let me add a comment